crates/rspack_binding_values/src/stats.rs

```diff
@@ -328,7 +328,7 @@ impl From<(String, rspack_core::LogType)> for JsStatsLogging {
         args: Some(vec![format!(
           "{}: {} ms",
           label,
-          secs * 1000 + subsec_nanos as u64 / 1000000
+          itoa::Buffer::new().format(secs * 1000 + subsec_nanos as u64 / 1000000)
```

Comment:
Can we have a util function for this, `itoa(xx)` instead of `itoa::Buffer::new().format(xx)`

Reply:
Of course, just a moment.